config/locales/candidate.yml

```diff
@@ -0,0 +1,13 @@
+en:
+  start:
+    heading: Apply for teacher training
+    what_is_it: Apply for teacher training is a new GOV.UK service being trialled with a small number of training providers.
+    what_you_will_need: What you’ll need
+    to_be_eligible: "To be eligible for teacher training, you need these qualifications (or their non-UK equivalents):"
+    a_degree: a degree
+    grade_4_english_maths: grade 4 (C) or above in English and maths GCSEs
+    grade_4_science: grade 4 (C) or above in GCSE science if you want to teach primary
+    before_you_start: Before you start a teacher training course you’ll also need to confirm you have the health and physical capacity to start training.
```

Comment:
should we group the items which are used in the radio button inside a `select:`?
```
  select:
    a_degree: a degree
    grade_4_english_maths: grade 4 (C) or above in English and maths GCSEs
    ...
```